src/ext/respToRedisLoader.c

```diff
@@ -40,29 +44,13 @@ struct RdbxRespToRedisLoader {
 static void onReadRepliesError(RdbxRespToRedisLoader *ctx) {
     RespReaderCtx *respReader = &ctx->respReader;
     int currIdx = ctx->respReader.countReplies % NUM_RECORDED_CMDS;
-    char *currCmdRecord = ctx->pendingCmds.cmdPrefix[currIdx];
-
-    /* Print also previous command if available. */
-    if (ctx->respReader.countReplies > 1) {
-        int prevIdx = (currIdx == 0) ? NUM_RECORDED_CMDS - 1 : currIdx - 1;
-        char *prevCmdRecord = ctx->pendingCmds.cmdPrefix[prevIdx];
-        RDB_reportError(ctx->p, (RdbRes) RDBX_ERR_RESP_WRITE,
-            "\nReceived Server error: \"%s\"\nGot failed on command [#%d] (First %d bytes):\n%s\n"
-            "\nPreceding command [#%d] was: \n%s\n",
-            respReader->errorMsg,
-            ctx->respReader.countReplies,
-            RECORDED_DATA_MAX_LEN,
-            currCmdRecord,
-            ctx->respReader.countReplies-1,
-            prevCmdRecord);
-    } else {
-        RDB_reportError(ctx->p, (RdbRes) RDBX_ERR_RESP_WRITE,
-            "\nReceived Server error:\n\"%s\"\n\nGot failed on command [#%d] (First %d bytes):\n%s\n",
-            respReader->errorMsg,
-            ctx->respReader.countReplies,
-            RECORDED_DATA_MAX_LEN,
-            currCmdRecord);
-    }
+
+    RDB_reportError(ctx->p, (RdbRes) RDBX_ERR_RESP_WRITE,
+                    "\nerror from dst '-%s' on key '%s' on command '%s' (RESP Command #%lu)\n",
```

Comment:
```suggestion
                    "\nerror from dst '-%s' on key '%s' on command '%s' (RESP Command #%zu)\n",
```
Seems like this is size_t.